package.json

```diff
@@ -14,6 +14,7 @@
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "^5.2.0",
     "@sentry/browser": "^5.15.5",
+    "@webpack-cli/serve": "^1.6.0",
```

Comment:
I'm not clear on why this is being added

Reply:
I had this issue when trying to run `./bin/webpack-dev-server`: https://github.com/rails/rails/issues/43062

Reply:
doesn't appear to be an issue after downgrading `webpack-dev-server` so I'll revert, but we'll probably have to bring it back once we update `@rails/webpacker` and `webpack-dev-server`

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1161/commits/c67959218a7a47d7c85a073afb70161b6ac0c5b1

Reply:
#1187 doesn't include all the webpacker dependency upgrades that this PR includes, but I think that's fine until webpacker itself is upgraded